tests/unit/cn/merge-test-cases/generated/crafted-cross_kvset_dups1.json

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
These files are generated by `make-merge-test-cases.py`.  Should update that script to produce JSON.

Reply:
Done, though that script and what it generates needs a lot more love. A task for another day, I guess